x/distribution/handler.go

```diff
@@ -13,38 +13,21 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 		switch msg := msg.(type) {
 		case types.MsgSetWithdrawAddress:
 			return handleMsgModifyWithdrawAddress(ctx, msg, k)
-		case types.MsgWithdrawDelegatorRewardsAll:
-			return handleMsgWithdrawDelegatorRewardsAll(ctx, msg, k)
 		case types.MsgWithdrawDelegatorReward:
 			return handleMsgWithdrawDelegatorReward(ctx, msg, k)
-		case types.MsgWithdrawValidatorRewardsAll:
-			return handleMsgWithdrawValidatorRewardsAll(ctx, msg, k)
+		case types.MsgWithdrawValidatorCommission:
+			return handleMsgWithdrawValidatorCommission(ctx, msg, k)
 		default:
 			return sdk.ErrTxDecode("invalid message parse in distribution module").Result()
 		}
 	}
 }
 
-//_____________________________________________________________________
-
-// These functions assume everything has been authenticated,
-// now we just perform action and save
+// These functions assume everything has been authenticated
```

Comment:
Can this specify what exactly the authentication expected is?

Reply:
Added comment.